### PR TITLE
Fix contribution guidelines

### PR DESCRIPTION
**Pull request by fmueller**
After successful integration of zappr this PR updates contribution guidelines according to this.

### CODE REVIEW

**Comment by whiskeysierra** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,5 @@
 ## Pull requests only
 
 **DON'T** push to the master branch directly. Always use pull requests and let people discuss changes in pull request.
-Together with the next point of the guidelines we can ensure to have very focused discussion that don't get off track
-as fast.
 
-Pull requests should only be merged after all discussions have been concluded and at least 1 reviewer gave his approval
+Pull requests should only be merged after all discussions have been concluded and at least 2 reviewer gave their approvals.
```
Comment (dehora) [inline CONTRIBUTING.md]: `2 reviewer gave their approvals.` -> `2 reviewers gave their approvals.`

**Comment by fmueller** [comment]
Added wording fixes @dehora @whiskeysierra

**Comment by ePaul** [comment]
:+1:

**Comment by ePaul** [comment]
:+1:

**Comment by whiskeysierra** [comment]
:+1:

**Pull request merged**